CSS_Tasks/CSS_Advanced/index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="bg">
+
+<head>
+    <meta content="text/html; charset=UTF-8" http-equiv="Content-Type" />
+    <title>WebGL</title>
+    <link href="stylesheet.css" rel="stylesheet">
+</head>
+
+<body>
+    <header class="header">
+        <button type="submit" class="button greenButton"> Вход </button>
+        <button type="submit" class="button greenButton"> Регистрация </button>
+    </header>
+
+    <main class="page-container">
+        <div class="navigation-bar">
+
+            <nav class="navigationBar">
+                <ul>
+                    <li>Компютърна графика с WebGL </li>
+                    <hr class="line">
+                    <li>Програмиране с Gо</li>
+                    <hr class="line">
+                    <li>Програмиране с Ruby</li>
+                    <hr class="line">
+                    <li>Agile</li>
+                    <hr class="line">
+                </ul>
+            </nav>
+
+        </div>
+        <div class="content">
+
+            <h1> Компютърна графика с WebGL</h1>
+            <hr class="line">
+
+            <h2>доц. П. Бойчев</h2>
+            <p>След завършване на курса студентите ще могат да създават тримерни статични изображения и тримерни анимации</p>
+
+            <p><i>Тип: </i> задължителна избираема</p>
+            <p><i>Курс: </i> 1 <sup>ви</sup></p>
+            <p><i>Програма: </i> бакалавър</p>
+
+            <textarea placeholder="Добавяне на коментар"></textarea>
+            <div class="mainButtons">
+                <button type="submit" class="button whiteButton"> Отказ </button>
+                <button type="submit" class="button greenButton"> Публикуване </button>
```

Comment:
Good job for combining the different classes, but maybe you can think for better names.